Realm.Shared/Schema/RealmSchema.cs

```diff
@@ -44,6 +56,12 @@ private RealmSchema(SchemaHandle handle, IEnumerable<ObjectSchema> objects)
             Handle = handle;
         }
 
+        /// <summary>
+        /// Find the definition of a class in this schema.
```

Comment:
Not sure what the convention at Realm is, but the .NET documentation usually describes what the method does from a 3rd person point of view, e.g. "[It] Finds the definition of a class in this schema."